### DISCUSSION TITLE
Install Keda with a specific spec

### DISCUSSION THREAD

**Discussion by nidiculageorge** [kedacore/charts#526] [Q&A] [answered]
We are installing keda and upgrading keda using the below commands.

Install
![image](https://github.com/kedacore/keda/assets/43850474/cf7b605a-93b0-42e4-af37-95e86ea5ff24)

Upgrade
![image](https://github.com/kedacore/keda/assets/43850474/7ae61753-bbf7-47c1-b4a9-00a17627934e)

Can we specify a spec for the keda while installing or upgrading using  helm like we need this much memory and cpu for keda pods

**Comment by tomkerkhove** [comment, marked as answer]
You can find all configuration options on https://artifacthub.io/packages/helm/kedacore/keda which also provides samples
